solutions/swb-ui/ui/.eslintrc.js

```diff
@@ -11,5 +16,16 @@ module.exports = {
       files: ['**/__tests__/**/*.[jt]s?(x)', '**/?(*.)+(spec|test).[jt]s?(x)'],
       extends: ['plugin:testing-library/react']
     }
-  ]
+  ],
+  settings: {
+    react: {
+      version: 'detect'
+    }
+  },
+  rules: {
+    // suppress errors for missing 'import React' in files
+    'react/react-in-jsx-scope': 'off',
```

Comment:
extend `plugin:react/jsx-runtime` instead

Reply:
yeah I did try that but it did not help much

Reply:
Let me give it a shot again !